decaylanguage/dec/dec.py

```diff
@@ -155,6 +155,14 @@ def parse(self, include_ccdecays=True):
         # At last, find all particle decays defined in the .dec decay file ...
         self._find_parsed_decays()
 
+        # Check whether certain decay model parameters are defined via
+        # variable names with actual values provided via 'Define' statements,
+        # and perform the replacements name -> value where relevant.
+        # Do also a replacement of 'a_float' with a_float.
```

Comment:
> Do also a replacement of 'a_float' with a_float.

Is this being done here? It's rather hidden if it is (it's fine if it is).

Reply:
Yep, it is.